projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Quotes.php

```diff
@@ -218,16 +209,22 @@ public function html( $quote, $metabox ) {
 								echo '<option value="' . esc_attr( $status_opt ) . '"' . esc_attr( $sel ) . '>' . esc_html( sprintf( __( '%s', 'zero-bs-crm' ), $status_str ) ) . '</option>'; // phpcs:ignore WordPress.WP.I18n.NoEmptyStrings
 							}
 							?>
-							</select>
-							
-						</div>
-						</td></tr>
+						</select>
+					</div>
+					<div class="jpcrm-form-group"></div>
 						<?php
 				} // end if
 
-                    #} ALSO customer assigning is seperate:
-                    ?><tr class="wh-large"><th><label><?php esc_html_e('Contact',"zero-bs-crm");?>:</label></th>
-                    <td><?php
+				?>
+						<style>
+							.zbstypeaheadwrap {
+								padding-top: 0px;
+								padding-bottom: 0px;
+							}
+						</style>
```

Comment:
Maybe add a bit of specificity and move to `sass/ZeroBSCRM.admin.quotebuilder.scss`?